test/graphql/types/Organization/creator.test.ts

```
@@ -0,0 +1,389 @@
+  const currentUserId = ctx.currentClient.user.id;
+
+  const currentUser = await ctx.drizzleClient.query.usersTable.findFirst({
+    columns: {
+      role: true,
+    },
+    with: {
+      organizationMembershipsWhereMember: {
+        columns: {
+          role: true,
+        },
+        where: (fields: { organizationId: any; }, operators: { eq: (arg0: any, arg1: string) => any; }) =>
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)